prisma/schema.prisma

```diff
@@ -322,25 +341,33 @@ model WalletLNbits {
   updatedAt  DateTime @default(now()) @updatedAt @map("updated_at")
   url        String
   invoiceKey String?
+  adminKeyId Int?     @unique
+  adminKey   Vault?   @relation(fields: [adminKeyId], references: [id])
 }
 
 model WalletNWC {
-  id         Int      @id @default(autoincrement())
-  walletId   Int      @unique
-  wallet     Wallet   @relation(fields: [walletId], references: [id], onDelete: Cascade)
-  createdAt  DateTime @default(now()) @map("created_at")
-  updatedAt  DateTime @default(now()) @updatedAt @map("updated_at")
-  nwcUrlRecv String?
-}
-
-model WalletBlink {
   id           Int      @id @default(autoincrement())
   walletId     Int      @unique
   wallet       Wallet   @relation(fields: [walletId], references: [id], onDelete: Cascade)
   createdAt    DateTime @default(now()) @map("created_at")
   updatedAt    DateTime @default(now()) @updatedAt @map("updated_at")
-  apiKeyRecv   String?
-  currencyRecv String?
+  nwcUrlRecv   String?
+  nwcUrlSendId Int?     @unique
+  nwcUrlSend   Vault?   @relation(fields: [nwcUrlSendId], references: [id])
```

Comment:
Oh, instead of `TODO_newName` in the wallet def, I could just use `nwcUrl` instead of `nwcUrlSend`. Then the new schema would match the existing typedef. Same for `WalletBlink`.